components/api.rst

```diff
@@ -75,20 +75,28 @@ Configuration variables:
 
 .. _api-actions:
 
-.. note::
-
-    Before a newly added ESPHome device can interact with the Home Assistant API it needs to be allowed to communicate
-    with it. This setting can be found in the ESPHome integration (NOT in the Add-On) by clicking "CONFIGURE" for
-    that device and enabling the "Allow device to make service calls" option.
-
 Actions
 -------
 
+Before using any of the actions below, you'll need to tell Home Assistant to allow your device to
+make service calls:
+
```

Comment:
We can also add a My button to link directly to the ESPHome integration list.

```suggestion

.. raw:: html

    <a href="https://my.home-assistant.io/redirect/integration/?domain=esphome" target="_blank" rel="noreferrer noopener"><img src="https://my.home-assistant.io/badges/integration.svg" alt="Open your Home Assistant instance and show an integration." /></a>


```

Reply:
Done.